src/otflow.jl

```diff
@@ -0,0 +1,98 @@
+struct OTFlow <: AbstractLuxLayer
+    d::Int          # Input dimension
+    m::Int          # Hidden dimension
+    r::Int          # Rank for low-rank approximation
+end
+
+# Constructor with default rank
+OTFlow(d::Int, m::Int; r::Int=min(10,d)) = OTFlow(d, m, r)
+
+# Initialize parameters and states
+function Lux.initialparameters(rng::AbstractRNG, l::OTFlow)
+    w = randn(rng, Float32, l.m) .* 0.01
+    A = randn(rng, Float32, l.r, l.d + 1) .* 0.01
+    b = zeros(Float32, l.d + 1)
+    c = zero(Float32)
+    K0 = randn(rng, Float32, l.m, l.d + 1) .* 0.01
+    K1 = randn(rng, Float32, l.m, l.m) .* 0.01
+    b0 = zeros(Float32, l.m)
+    b1 = zeros(Float32, l.m)
+    
+    ps = (w=w, A=A, b=b, c=c, K0=K0, K1=K1, b0=b0, b1=b1)
+    st = NamedTuple()
+    return ps, st
```

Comment:
```suggestion
    return (; w, A, b, c, K0, K1, b0, b1)
```

`initialparameters` shouldn't return the states